lib/components/programs-list/index.tsx

```diff
@@ -19,42 +29,69 @@ const ProgramListing: React.FC<ProgramListingProps> = ({
-        <Heading textAlign={useMobileLayout ? 'left' : 'right'}>{date}</Heading>
-        <Text>{time}</Text>
-      </Flex>
-      <Box
-        h={useMobileLayout ? '4px' : 'maxContent'}
-        minWidth={useMobileLayout ? 'maxContent' : '4px'}
-        bg={'var(--magenta)'}
-      />
-      <Flex flexDirection="column" alignItems="start">
-        <Heading>{title}</Heading>
-        <Text fontSize="lg" whiteSpace="pre-line">
-          {description}
-        </Text>
-        <Link
-          href={recordingLink || registerLink}
-          isExternal
-          _hover={{ textDecoration: 'none' }}
-          _focus={{ outline: 'none' }}>
+    <>
+      <EditProgramModal modalManagement={editProgramModalManagement} />
+
+      <Flex flexDirection={useMobileLayout ? 'column' : 'row'} gap={4}>
+        <Flex
+          flexDirection="column"
+          alignItems={useMobileLayout ? 'start' : 'end'}
+          minWidth={'40%'}>
+          <Heading textAlign={useMobileLayout ? 'left' : 'right'}>{date}</Heading>
+          <Text>{time}</Text>
+        </Flex>
+        <Box
+          h={useMobileLayout ? '4px' : 'maxContent'}
+          minWidth={useMobileLayout ? 'maxContent' : '4px'}
+          bg={'var(--magenta)'}
+        />
+        <Flex flexDirection="column" alignItems="start">
+          <Heading>{title}</Heading>
+          <Text fontSize="lg" whiteSpace="pre-line">
+            {description}
+          </Text>
+          <Link
+            href={recordingLink || registerLink}
+            isExternal
+            _hover={{ textDecoration: 'none' }}
+            _focus={{ outline: 'none' }}></Link>
           <Button
             mt="5px"
-            bg="var(--magenta)"
+            padding="15px 30px"
             borderRadius="md"
-            alt="External page link to register for this event."
-            pointerEvents="none">
-            <Text margin="2px" color="white" fontSize="large" fontWeight="bolder">
-              {recordingLink ? `VIEW RECORDING` : `REGISTER HERE`}
+            alt="Button to edit this event"
+            variant="osl"
+            _hover={{ textDecoration: 'none' }}
+            _focus={{ outline: 'none' }}
+            onClick={() => editProgramModalManagement.onOpen()}>
+            <Text margin="2px" fontSize="large" fontWeight="bolder">
+              EDIT EVENT
             </Text>
-          </Button>
-        </Link>
+          </Button>{' '}
+          :
+          <Link
+            href={recordingLink || registerLink}
+            isExternal
+            _hover={{ textDecoration: 'none' }}
+            _focus={{ outline: 'none' }}>
+            <Button
+              mt="5px"
+              bg="var(--magenta)"
+              borderRadius="md"
+              alt="External page link to register for this event."
+              pointerEvents="none">
+              <Text margin="2px" color="white" fontSize="large" fontWeight="bolder">
```

Comment:
it was mentioned that refactoring this is out of the scope of the ticket, but for the future is there a way to abstract the styling outside of the code for this component?

Reply:
Yeah so only commit changes to the files you've changed. If you let the linter run and do git add . it will comingle your changes with other files.
tl;dr: don't do git add . or git commit -a unless you've checked all changes are yours